### PR TITLE
Final Milestone: Basic functionality

### PR DESCRIPTION
**Pull request by SarvarKh**
# Library project - Basic functionality
In this milestone, we have successfully
- added basic HTML elements (form and table) and styled them;
- added function to get user input from form HTML;
- added function to create book obj;
- added created book to the library array;
- added function to display added books with 2 btns;
- 1st btn: change "read" status;
- 2nd btn: remove the book;

### CODE REVIEW

**Comment by MuriloRoque** [comment]
## Status: Invalid :red_circle: 

### Hello @SarvarKh and @apricot12  :wave: 

### Unfortunately I need to invalidate your submission because of the following issues:

- [ ] Your README file contains only the title of the project, please build a more professional one before requesting for a new review.

Kindly make the requested changes and submit them for another review, please :pray: 

I will be on Slack 24/7 in case you have any questions about this review :handshake: 

Murilo Roque Paiva da Silva

- Github: [@muriloroque](https://github.com/MuriloRoque)
- Linkedin: [MuriloRoque](https://www.linkedin.com/in/murilo-roque-b1268741/)

[Twitter Follow](https://twitter.com/MuriloRoquePai1)

**Review by mikenath223 — CHANGES_REQUESTED** [review]
### Hello Team 👋

**Good job on this. Well written readme and a clean, easy-to-understand code 👌**

However, let's make a couple of changes.

- [x] Please deploy the project online and include the live deploy link in the readme.

- Kindly refer to other comments on the code.

### **STATUS: CHANGES REQUESTED** 👷‍♂️

Kindly fix these issues and request another review👍
Feel free to contact me on slack(michgolden) if you need further clarification.

**HAPPY CODING 🎉💻🎉**

**Michgolden Ukeje (CR)**

File: scripts/main.js [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+  const td1 = document.createElement('td');
+  const td2 = document.createElement('td');
+  const td3 = document.createElement('td');
+  const td4 = document.createElement('td');
+  const td5 = document.createElement('td');
+  const removeBtn = document.createElement('button');
+  const td4Btn = document.createElement('button');
+
+  td1.textContent = newBook.author;
+  td2.textContent = newBook.title;
```
Comment (mikenath223) [inline scripts/main.js]: - [x] Apart from the constructor function kindly change these two functions to an es6 arrow function syntax.

**Comment by mikenath223** [comment]
- [x] Optional - Consider adding form validations to prevent an empty book from being created.

**Comment by mikenath223** [comment]
- [ ] Optional - Also consider including localStorage to save the books for data persistence.

File: scripts/main.js (hunk 1 above)
Comment (SarvarKh) [reply scripts/main.js]: Done

**Review by andresporras3423 — APPROVED** [review]
# Status: Approved :heavy_check_mark: :heavy_check_mark: :heavy_check_mark:
**Hello Sarvar Khalimov & Nihal Mohammed** 
Amazing job :clap: :clap: you have met all the requirements for this project
Now you are ready to continue with your next challenge :muscle: 
Don't forget to merge with your development branch
 
Keep going like this,
 
Oscar Andrés Russi Porras

**Pull request merged**